src/common/PersonTile/index.js

```diff
@@ -0,0 +1,11 @@
+import { Tile, Image, Name, Role } from "./styled";
+
+export const PersonTile = ({ name, role }) => {
+  return (
+    <Tile>
+      <Image src="https://i.ibb.co/QHf40YG/poster-1.png" alt="" />
+      <Name>{name ? name : "test name"}</Name>
+      <Role>{role ? role : "test role"}</Role>
```

Comment:
Moim zdaniem nie trzeba pisać warunku. Jeżeli nie przekażesz propsa, będzie on pusty a co za tym idzie nic nie wyświetli.

Reply:
Jak nie przekażę propsa to będzie samo zdj wyświetlane, a testowo myślę, że może być tam cokolwiek pod tym zdjęciem :D